docs/source/tutorials/quickstart.rst

```diff
@@ -180,18 +180,18 @@ The output should look as follows.
            self.play(FadeOut(square))           # fade out animation
 
 This example shows one of the most basic features of manim: the ability to
-implement complicated and mathematically-intensive animations (such as cleanly
+implement complicated and mathematically intensive animations (such as cleanly
 interpolating between two geometric shapes) in very few lines of code.
 
 
 ************
 You're done!
 ************
 
-With a working installation of manim, and the bare basics under your belt, it
+With a working installation of manim and the bare basics under your belt, it
 is now time to start creating awesome mathematical animations.  For a look
 under the hood at what manim is doing when rendering the ``SquareToCircle``
-scene, see the next tutorial :doc:`a_deeper_look`.  For an extensive review of
+the scene, see the next tutorial :doc:`a_deeper_look`.  For an extensive review of
```

Comment:
Original is correct, since a_deeper_look covers the Scene class in general and not this particular scene.

Reply:
Also, is grammatically incorrect: `the SquareToCircle the scene` doesn't make any sense, `the SquareToCircle scene` does.

Reply:
Chnaged